components/bl-clock/src/stopwatch.js

```diff
@@ -33,10 +62,27 @@ export function Stopwatch({ component }) {
   }, []);
 
   return (
-    <div className="stopwatch"> { remainingSecond } </div>
+    <>
+      {hasHours(stopwatchTimeFormat) && (<span className="stopwatch">{`${pad(time.hours)}:`}</span>)}
+      {hasMinutes(stopwatchTimeFormat) && (<span className="stopwatch">{`${pad(time.minutes)}:`}</span>)}
```

Comment:
duplicated calculations

create use memo that only once will parse time format and creates map with tokens like

```
FormatTokens = {
 HH: true,
 MM: false
}
```